### ISSUE TITLE
Missing CFBundleVersion when uploading to ITMS

### ISSUE THREAD

**Issue by Patrick-Remy** [mw99/DataCompression#20]
When using carthage as a dependency manager, i receive following Error if I try to upload my build

```
ERROR ITMS-90056: "This bundle Payload/..../Frameworks/DataCompression.framework is invalid. The Info.plist file is missing the required key: CFBundleVersion. Please find more information about CFBundleVersion at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion"
Return status of iTunes Transporter was 1: ERROR ITMS-90056: "This bundle Payload/..../Frameworks/DataCompression.framework is invalid. The Info.plist file is missing the required key: CFBundleVersion. Please find more information about CFBundleVersion at https://developer.apple.com/documentation/bundleresources/information_property_list/cfbundleversion"
The call to the iTMSTransporter completed with a non-zero exit status: 1. This indicates a failure.
```

**Comment by mw99** [comment]
nkay~ will take look when I get some free time

**Comment by mw99** [comment]
Sorry this took forever. I assume this is fixed in version [**3.6.0**](https://github.com/mw99/DataCompression/releases/tag/3.6.0)
